### PR TITLE
Affiche le montant de l'AAH dans le cadre de l'expérimentation

### PR DESCRIPTION
**Pull request by guillett**
Please @alexsegura have a look but do not merge.

### CODE REVIEW

File: app/styles/resultat.css [hunk 1]
```diff
@@ -23,10 +23,6 @@
     font-size: 80%;
 }
 
-#simulationId {
```
Comment (alexsegura) [inline app/styles/resultat.css]: It is also used in:
- `app/views/partials/foyer/recap-situation.html`
- `app/views/partials/trace.html`

File: app/js/services/resultatService.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ angular.module('ddsApp').service('ResultatService', function($http, droitsDescri
     *@return   {Object}  A new object containing the ressources of the family and of the individual. The family ressources will be overridden if conflicting.
     */
     function normalizeOpenfiscaRessources(testCase) {
-        return _.merge({}, testCase.menages._, testCase.familles._, testCase.individus.demandeur);
+        var individuId = testCase.menages._.personne_de_reference[0];
```
Comment (alexsegura) [inline app/js/services/resultatService.js]: Is there always at least 1 element in the array?

Comment (guillett) [reply app/js/services/resultatService.js]: Well, yes, in all valid situation. Worst case scenario, arror should be capture in the catch section.

**Comment by alexsegura** [comment]
:shipit:

**Pull request merged**